kafka-ui-react-app/src/components/Topics/Topic/Details/Details.tsx

```diff
@@ -19,13 +19,15 @@ import SettingsContainer from './Settings/SettingsContainer';
 interface Props extends Topic, TopicDetails {
   clusterName: ClusterName;
   topicName: TopicName;
+  isInternal: boolean | undefined;
```

Comment:
I think this can be `boolean` only, since you're making `!!internal` in the selector.

Reply:
done